src/thing.cpp

```diff
@@ -17,8 +17,6 @@
  * 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
  */
 
-#include "otpch.h"
```

Comment:
But if it is included on at least most of the files, it will be indirectly included on all of them since headers depend on other headers.